index.html

```diff
@@ -229,19 +229,37 @@
 <body>
 
   <div id="main-container" class="container-fluid">
-    <div class="row">
-      <div class="col-3">
-
-        <nav id="left-nav" role="navigation">
-          <ul id="savedConfig" class="list-group">
-          </ul>
-          <button class="btn btn-primary btn-block" type="submit" id="saveConfigBtn" disabled>Save</button>
-        </nav>
-
-
-      </div>
-      <div class="col-9">
-
+    <div class="row p-5">
+        <div class="col-md-11 mx-auto mb-5">
+        <!-- TradingView Widget BEGIN -->
+        <div class="tradingview-widget-container" style="height:300px;">
+            <div id="tradingview_6d876"></div>
+            <div class="tradingview-widget-copyright"><a href="https://www.tradingview.com/symbols/COINBASE-BTCUSD/" rel="noopener" target="_blank"><span class="blue-text">BTCUSD chart</span></a> by TradingView</div>
```

Comment:
Why do we have `COINBASE-BTCUSD` and later `KRAKEN:XBTUSD`?
On top of that, this website is not only about Bitcoin, but any digital assert available through Kraken.